Bottom: ffcad96833159616f61391b7abbe6e36c874344e
Top:    d289d3c245db89cba658d7060a02dfd27b269281
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-23 18:52:57 +0000

Refresh of add-a-hacky-name-mangling-thin

---

diff --git a/doc/pcbfile.texi b/doc/pcbfile.texi
index 7ec8517..7dd6be8 100644
--- a/doc/pcbfile.texi
+++ b/doc/pcbfile.texi
@@ -881,11 +881,9 @@ Marker used internally to avoid revisiting an object.
 @item 0x10000 nopaste
 For pads, set to prevent a solderpaste stencil opening for the
 pad.  Primarily used for pads used as fiducials.
-@item 0x20000 striphierarchy
-Strip hierarchy when displaying this element's refdes.
 @end table
 @c pcbfile ~pcbflags
-@c ./../src/const.h 140
+@c ./../src/const.h 139
 @node PCBFlags
 @section PCBFlags
 @table @code
